accounts_acc_test.go

```diff
@@ -110,9 +110,17 @@ var _ = Describe("Accounts", func() {
 			if errTP := client.AccountTeamProjects.Create(
 				account.Account.Id,
 				team.Team.Id,
-				AccountTeamProject{ProjectName: projectName, TeamType: projectType}); errTP != nil {
+				AccountTeamProject{ProjectName: projectName, TeamType: "developer"}); errTP != nil {
 				Fail("cannot create account team project association:" + errTP.Error())
 			}
+
+			By("Update account team project")
+			if errTPu := client.AccountTeamProjects.Update(
```

Comment:
AFAIR `errTPu` is not a go-style. `err` is enough(I saw that the other code uses this which is still not so correct)

Reply:
Here in the acceptance tests, we do have many errors, and I did add an explicit postfix to an error and clearly see where this error is coming from.

No "go style" manifest or linter forbids naming errors differently from `err`, so I prefer to leave it this way for readability reasons.